src/main/java/helpers/DataProviders.java

```diff
@@ -1,15 +1,38 @@
 package helpers;
 
 import org.testng.annotations.DataProvider;
+import ru.yandex.qatools.allure.annotations.Step;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
 
 public class DataProviders {
+
     private static final String REGISTRATION_DATA_FILE = "/RegistrationData.csv";
+    private static final Path AUTHENTICATION_DATA_FILE = Paths.get("./src/main/resources/LoginExamples.csv") ;
 
     @DataProvider(name = "registrationData")
     public Object[][] getRegistrationData() {
-        return new Object[][] {
+        return new Object[][]{
                 {},
                 {}
         };
     }
+
+    @DataProvider(name = "loginData")
+    public static Object[][] getAuthenticationData() throws IOException {
+        return getData( AUTHENTICATION_DATA_FILE,"," );
+    }
+
+    public static Object[][] getData(Path path, String delimetr) throws IOException {
+        List<String> data = Helpers.readAllLines(path);
+        Object[][] dataRows = new Object[data.size()][];
+        for (int i = 0; i < data.size(); i++) {
+            dataRows[i] = data.get(i).split(delimetr);
```

Comment:
add a private method here that will convert " " -> "" in String [].
let its signature be private String[] convertSpaceToEmptyString(String[] values)